addons/hr_skills/static/tests/many2one_avatar_employee.test.js

```diff
@@ -0,0 +1,98 @@
+import {
+    click,
+    contains,
+    start,
+    startServer,
+} from "@mail/../tests/mail_test_helpers";
+import { mountView } from "@web/../tests/web_test_helpers";
+import { beforeEach, describe, expect, test } from "@odoo/hoot";
+import { queryAttribute } from "@odoo/hoot-dom";
+import { defineHrSkillModels } from "@hr_skills/../tests/hr_skills_test_helpers";
+
+describe.current.tags("desktop");
+defineHrSkillModels();
+
+let data = {};
+beforeEach(async () => {
+    const pyEnv = await startServer();
+
+    // Skills
+    [data.skillJavaId, data.skillTigrinya] = pyEnv["hr.skill"].create([
+        {
+            name: "Java",
+        },
+        {
+            name: "Tigrinya",
+        },
+    ]);
+
+    // User
+    data.partnerPierreId = pyEnv["res.partner"].create({
+        name: "Pierre",
+    });
+    data.userPierreId = pyEnv["res.users"].create({
+        name: "Pierre",
+        partner_id: data.partnerPierreId,
+    });
+
+    // Employees
+    data.employeePierreId = pyEnv["hr.employee"].create({
+        name: "Pierre",
+        user_id: data.userPierreId,
+        user_partner_id: data.partnerPierreId,
+    });
+
+    // Employee skills
+    [data.employeeSkill1Id, data.employeeSkill2Id] = pyEnv["hr.employee.skill"].create([
+        {
+            employee_id: data.employeePierreId,
+            skill_id: data.skillJavaId,
+        },
+        {
+            employee_id: data.employeePierreId,
+            skill_id: data.skillTigrinyaId,
+        },
+    ]);
+
+    // Public employee
+    // Imitating the server behavior by creating an hr.employee.public record with the same data and same id
+    pyEnv["hr.employee.public"].create({
+        name: "Pierre",
+        employee_skill_ids: [data.employeeSkill1Id, data.employeeSkill2Id],
+    });
+
+    // Fake record
+    data.recordPierreId = pyEnv["m2o.avatar.employee"].create([
+        { employee_id: data.employeePierreId },
+    ]);
+});
+
+test("many2one_avatar_employee widget in kanban view with skills on avatar card", async () => {
+    await start();
+    await mountView({
+        type: "kanban",
+        resModel:"m2o.avatar.employee",
+        arch: `<kanban>
+            <templates>
+                <t t-name="kanban-box">
+                    <div>
+                        <field name="employee_id" widget="many2one_avatar_employee"/>
+                    </div>
+                </t>
+            </templates>
+        </kanban>`,
+    });
+
+    await contains(".o_m2o_avatar", { count: 1 });
+
+    // Kanban card should display employee avatar
+    await contains(".o_field_many2one_avatar_employee img", { count: 1 });
+    expect(
+        queryAttribute(".o_kanban_record .o_field_many2one_avatar_employee img", "data-src")
+    ).toBe(`/web/image/hr.employee/${data.employeePierreId}/avatar_128`);
```

Comment:
Comment is useless, the assertion of `data-src` is obviously doing that. Everyone knows `img [src=url]` will show an image with `url`. And the url has semantics of pierre employee's avatar.